src/PinI2C.h

```diff
@@ -82,5 +82,26 @@ class PinI2C
 
 };
 
+class PinI2C_LoopBack
+{
+public:
+  
+  /*!
+   * @brief Constructor.
+   * @param w Wrapper object representing the target the pin is connected to.
+   */
+  PinI2C_LoopBack(I2Cwrapper* w) {}
+
+  void pinMode(uint8_t pin, uint8_t mode) { ::pinMode(pin,mode); };
+  void digitalWrite(uint8_t pin, uint8_t value) { ::digitalWrite(pin,value); };
+  int digitalRead(uint8_t pin) { return ::digitalRead(pin); };
+  int analogRead(uint8_t pin) { return ::analogRead(pin); };
+  void analogReference(eAnalogReference mode) { ::analogReference(mode); };
+  void analogWrite(uint8_t pin, int value) { ::analogWrite(pin,value); };
+  
+private:
+
+};
```

Comment:
I thought about other ways of doing this:

*  Put the `#ifdef` in the class `PinI2C` so no extra class is required. But that would require `#ifdef`'s in the .cpp, and I get confused about the scope of a `#define`, especially with the arduino ide.

```
#define I2CWRAPPER_LOOPBACK
#include <PinI2C.h>
PinI2C pins(&wrapper); // no change here